index.js

```diff
@@ -710,7 +686,7 @@ RFID.prototype.stopListening = function (callback) {
 };
 
 RFID.prototype.disable = function () {
-  this.irq.removeListener('fall', this.irqcallback);
+  this.irq.removeAllListeners();
```

Comment:
We don't use the `fall` IRQ anymore so just remove any remaining listeners (should be all `low`).